README.md

```diff
@@ -71,6 +71,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Cotiviti Nepal](http://cotiviti.com.np) | Hattisar, Kathmandu | Software research and development center
 - [Crystal Infosys](http://crystalinfosys.com/) | Jadibuti, Kathmandu | Web Development, Internet Marketing SEO
 - [Curves n' Colors](http://curvesncolors.com/) | Kamaladi, Kathmandu | Design, Copywriting, Branding
+- [Crimson Tech](http://crimsontech.io/) | Balkumari, Lalitpur | Engineering Solution specialization on Computer vison, AI/ML, IOT, Software Development, Industrial Automation
```

Comment:
@yubarajsingh Thank you for contributing. Can you put it in order? The list is sorted alphabetically.